File: src/main/scala/committee/nova/quit/client/key/KeyInit.scala
package committee.nova.quit.client.key

import cpw.mods.fml.client.registry.ClientRegistry
import net.minecraft.client.settings.KeyBinding
import org.lwjgl.input.Keyboard

object KeyInit {
  val fastQuit = new KeyBinding("key.quit.fastQuit", Keyboard.KEY_X, "key.categories.quit")
  val bossKey = new KeyBinding("key.quit.bossKey", Keyboard.KEY_B, "key.categories.quit")

  def init(): Unit = {
    ClientRegistry.registerKeyBinding(fastQuit)
    ClientRegistry.registerKeyBinding(bossKey)
  }
}

File: src/main/java/committee/nova/quit/client/key/KeyInit.java
package committee.nova.quit.client.key;

import net.minecraft.client.settings.KeyBinding;
import net.minecraftforge.client.settings.KeyConflictContext;
import net.minecraftforge.fml.client.registry.ClientRegistry;
import org.lwjgl.input.Keyboard;

public class KeyInit {
    public static final KeyBinding fastQuit = new KeyBinding(
            "key.quit.fastQuit",
            KeyConflictContext.GUI, Keyboard.KEY_Q,
            "key.categories.quit"
    );
    public static final KeyBinding bossKey = new KeyBinding(
            "key.quit.bossKey",
            KeyConflictContext.GUI, Keyboard.KEY_B,
            "key.categories.quit"
    );

    public static void init() {
        ClientRegistry.registerKeyBinding(fastQuit);
        ClientRegistry.registerKeyBinding(bossKey);
    }
}
